httpclient5/src/main/java/org/apache/hc/client5/http/impl/classic/ProxyClient.java

```diff
@@ -140,20 +140,19 @@ public Socket tunnel(
         Args.notNull(proxy, "Proxy host");
         Args.notNull(target, "Target host");
         Args.notNull(credentials, "Credentials");
-        HttpHost host = target;
-        if (host.getPort() <= 0) {
-            host = new HttpHost(host.getSchemeName(), host.getHostName(), 80);
+        if (target.getPort() <= 0) {
```

Comment:
@arturobernalg It would be nicer (and more consistent) to use `Args#check` here but not a deal breaker.

Reply:
@ok2c  
True. changed.